models/stations.go

```diff
@@ -157,3 +157,22 @@ type UseSchema struct {
 	StationName string `json:"station_name" binding:"required"`
 	SchemaName  string `json:"schema_name" binding:"required"`
 }
+
+type RemoveSchemaFromStation struct {
+	StationName string `json:"station_name" binding:"required"`
+}
+
+type SchemaStation struct {
+	SchemaName    string `json:"name" bson:"name"`
+	VersionNumber int    `json:"version_number" bson:"version_number"`
+}
+
+type SchemaDetails struct {
```

Comment:
this type is not needed you don't need to save in the DB the state of the schema whther is active or not, here at least so please delete this type